go.mod

```diff
@@ -154,3 +170,5 @@ toolchain go1.21.0
 replace github.com/jsipprell/keyctl => github.com/hallyn/keyctl v1.0.4-0.20230720164202-b9476cd969e8
```

Comment:
Let's fork the project into project-machine if we are forking and using forks. Avoid personal forks as much as possible. Of course, I am commenting on this without context, so just consider this as a preference comment.

Reply:
We really don't want a fork.  Actually we probably don't need this fork any more - i think early go mod versions didn't handle this as well, but as my PR is merged, but there simply isn't a release with it, we can probably just replace by git commit from the original repo.